roles/quality.md

```diff
@@ -1,10 +1,10 @@
 # Codebase Steward, Quality
 
-The Foundation For Public Code helps developers of Open Source software used for public uses – such as in governments and cities – make their codebases high quality so that they are understandable, well documented, easy to work with and highly reusable.
+The Foundation For Public Code helps the developers of Open Source software for public use – such as in governments and cities – improve the quality of their codebases by making them easier to understand, well documented, and thus highly reusable. 
 
-We provide review of contributions to Codebases that are in Codebase Stewardship at the Foundation For Public Code in order to help developers build trusted and reliable software that can be widely reused and can foster a strong community. We test against the quality Public Code standards for code and documentation we set which aim to make Codebases understandable, reusable and community based. Our standards are high as well as opinionated to give trust the code can really serve as the infrastructure of society as well as continually improving with the changing world whilst keeping the choices of technology and architecture open.
+We provide review of contributions to Codebases that are in Codebase Stewardship at the Foundation For Public Code in order to help developers build trusted and reliable software that can be widely reused and can foster a strong community. We test against the quality Public Code standards for code and documentation we set which aim to make Codebases understandable, reusable and community based. Our standards are high, as well as opinionated, to ensure Codebases can be trusted as the infrastructure of society, improve continually, and keep technology and architectural choices open.
 
-As a Codebase Steward, Quality for Codebases in our Stewardship you:
```

Comment:
I don't actually think this increases readability and introduces a wildly more complex sentence structure. 

The comma in the title really doesn't help here as well.

Perhaps change this to:

> As a Codebase Steward for Quality you help Codebases in our Stewardship by:

This applies to #95, #96 and #97 as well.